integration-tests/cli/csv-parser-price.it.js

```diff
@@ -22,123 +22,105 @@ describe('CSV and CLI Tests', () => {
   const samplesFolder = './packages/csv-parser-price/test/helpers/'
   const binPath = './integration-tests/node_modules/.bin/csvparserprice'
   let apiConfig
-  beforeAll(() =>
-    getCredentials(projectKey).then(credentials => {
-      apiConfig = {
-        host: CONSTANTS.host.auth,
-        apiUrl: CONSTANTS.host.api,
-        projectKey,
-        credentials: {
-          clientId: credentials.clientId,
-          clientSecret: credentials.clientSecret,
-        },
-      }
-    })
-  )
+  beforeAll(async () => {
+    const credentials = await getCredentials(projectKey)
+    apiConfig = {
+      host: 'https://auth.sphere.io',
+      apiUrl: 'https://api.sphere.io',
+      projectKey,
+      credentials,
+    }
+  }, 10000)
 
   describe('CLI basic functionality', () => {
-    it('should print usage information given the help flag', done => {
-      exec(`${binPath} --help`, (error, stdout, stderr) => {
-        expect(String(stdout)).toMatch(/help/)
-        expect(error && stderr).toBeFalsy()
-        done()
-      })
+    it('should print usage information given the help flag', async () => {
+      const [stdout, stderr] = await exec(`${binPath} --help`)
+      expect(stderr).toBeFalsy()
+      expect(stdout).toMatchSnapshot()
     })
 
-    it('should print the module version given the version flag', done => {
-      exec(`${binPath} --version`, (error, stdout, stderr) => {
-        expect(stdout).toBe(`${version}\n`)
-        expect(error && stderr).toBeFalsy()
-        done()
-      })
+    it('should print the module version given the version flag', async () => {
+      const [stdout, stderr] = await exec(`${binPath} --version`)
+      expect(stderr).toBeFalsy()
+      expect(stdout).toBe(`${version}\n`)
     })
 
-    it('should write output to file', done => {
+    it('should write output to file', async () => {
       const csvFilePath = path.join(samplesFolder, 'simple-sample.csv')
       const jsonFilePath = tmp.fileSync().name
 
-      // eslint-disable-next-line max-len
-      exec(
-        `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`,
-        (cliError, stdout, stderr) => {
-          expect(cliError && stderr).toBeFalsy()
-
-          fs.readFile(jsonFilePath, { encoding: 'utf8' }, (error, data) => {
-            expect(data.match(/prices/)).toBeTruthy()
-            expect(error).toBeFalsy()
-            done()
-          })
-        }
+      await exec(
+        `${binPath} -p ${projectKey} -i ${csvFilePath} -o ${jsonFilePath}`
       )
+
+      const data = await fs.readFile(jsonFilePath, { encoding: 'utf8' })
+      expect(data).toMatchSnapshot()
```

Comment:
You can do this instead
```js
await expect(fs.readFile(jsonFilePath, { encoding: 'utf8' })).resolves.toMatchSnapshot()
```

Reply:
I think this gets a bit too long for one line. I also saw that prettier splits it into three lines instead.